Zend/zend_language_scanner.l

```diff
@@ -631,6 +634,27 @@ zend_op_array *compile_filename(int type, zval *filename)
 		convert_to_string(&tmp);
 		filename = &tmp;
 	}
+	if (!CG(script_extensions)[0]) {
```

Comment:
@smalyshev We already have "allow_url_include=Off" by default. I'm not going to change it. Isn't "allow_url_include=Off" enough?